TARS/src/mcu_main/error.cpp

```diff
@@ -0,0 +1,139 @@
+
+/**
+ * If an error during initialization was detected, some combination of the blue, green, and orange LEDs will be on,
+ * and the red LED will flash at some frequency. The Blue, Green, and Orange LEDs tells you what sensor is errored.
+ *
+ *
+ * GREEN => KX134
+ * GREEN, BLUE => GPS
+ * ORANGE => SD
+ * RED, BLUE, ORANGE => RADIO
+ * BLUE => MAGNETOMETER
+ * BLUE, ORANGE => BNO,
+ * NONE (and RED is flashing) => Something is fucked, call whoever wrote this code
+ *
+ * RED, 10 Hz => Connection Error
+ * RED, 2 Hz => Initialization Error
```

Comment:
Update to something more easily distinguishable. 
10 Hz even blinking and 2 fast blinks followed by paused, maybe?